services/configuration/doc/testnet-api-tutorial.md

```diff
@@ -179,7 +179,7 @@ endpoint.
        `validators` field, as specified in `from` field of propose
```

Comment:
Please correct the paragraph as follows:
1. \*actual* config contains the node-sender public key in the \`validators\` field array, as specified in \`from\` field of the propose transaction. The \`from\` field is determined by the public key of the node which \`postpropose\` endpoint is accessed for signing the transaction on maintainter's behalf.	
as "maintainer" is an animate object "'s" is allowed here, while "node" is not so in the latter case it is preferred to avoid "'s" forms